cli/test/README.md

```diff
@@ -93,8 +93,9 @@ This document includes CLI commands and the expected CLI output in order to help
 |`fastddsqosprof file.xml set participant[profile].builtin.external_locators.metatraffic_unicast[] arg1 arg2 arg3 arg4 arg5 arg6`|**UNSUPPORTED** (currently `Fast DDS QoS Profiles Manager exception caught: Unsupported`)|N/A|
 |`fastddsqosprof file.xml set participant[profile].builtin.external_locators.metatraffic_unicast[].address`|`ERROR: set command for Participant builtin <external_locators> expects at least 1 arguments and received 0`|`PARTICIPANT_BUILTIN_METATRAFFIC_EXTERNAL_UNICAST_LOCATORS_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].builtin.external_locators.metatraffic_unicast[].address -h`|N/A|`PARTICIPANT_BUILTIN_METATRAFFIC_EXTERNAL_UNICAST_LOCATORS_USAGE`|
-|`fastddsqosprof file.xml set participant[profile].builtin.external_locators.metatraffic_unicast[].address argument` [^2]|**UNSUPPORTED** (currently `Fast DDS QoS Profiles Manager exception caught: Unsupported`)|N/A|
-|`fastddsqosprof file.xml set participant[profile].builtin.external_locators.metatraffic_unicast[11811].address argument`|**UNSUPPORTED** (currently `Fast DDS QoS Profiles Manager exception caught: Unsupported`)|N/A|
+|`fastddsqosprof file.xml set participant[profile].builtin.external_locators.metatraffic_unicast[].address argument` [^2]|N/A|N/A|
```

Comment:
Why is `[^2]` added to a `set` command?
Isn't it refer only to `print`, `clear` and `query`?

Reply:
That is a footnote just to remember that in those verbs, an additional validity check will be needed. It is a TODO in the test plan, so to say.

Reply:
Nice!